include/lsst/afw/geom/EdgeRolloffFunctor.h

```diff
@@ -0,0 +1,67 @@
+// -*- lsst-c++ -*-
+
+/* 
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ * 
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/**
+ * \file 
+ * @brief Functor to describe edge rolloff effect in CCDs.
+ */
+
+#ifndef LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+#define LSST_AFW_GEOM_EDGEROLLOFFFUNCTOR_H
+
+#include "lsst/afw/geom/Functor.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+
+/// @brief Stubbs parameterization of edge rolloff effect in LSST CCDs.
+class EdgeRolloffFunctor : public lsst::afw::geom::Functor {
+
+public:
+
+   EdgeRolloffFunctor(std::string const & owner,
+                      double amplitude, double xscale, double xmax);
+
+   ~EdgeRolloffFunctor() {}
+
+   virtual PTR(Functor) clone() const;
+
+   virtual double operator()(double x) const;
+
+   virtual double deriv(double x) const;
```

Comment:
Should this be a public method (is it likely to be useful by others) or can it be protected (in which case it should have a leading underscore in the name).

In any case I suggest expanding the name to derivative, since our naming standard frowns on abbreviations.